internal/criocli/criocli.go

```diff
@@ -738,6 +742,14 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			EnvVars:   []string{"CONTAINER_GLOBAL_AUTH_FILE"},
 			TakesFile: true,
 		},
+		&cli.StringFlag{
+			Name:      "namespaced-auth-dir",
+			Usage:     "Path to the root directory for namespaced auth files. Must be an absolute path.",
+			Value:     defConf.NamespacedAuthDir,
+			EnvVars:   []string{"CONTAINER_NAMESPACED_AUTH_DIR"},
+			TakesFile: true,
+			Hidden:    true,
```

Comment:
I think it's important to hide this option but allow setting it for testing purposes.

Reply:
I think we want to remove it from completions as well, and we not want to write a release note about it, maybe?